src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -394,7 +394,8 @@
     <value>Analyzer Failure</value>
   </data>
   <data name="CompilerAnalyzerThrows" xml:space="preserve">
-    <value>Analyzer '{0}' threw an exception of type '{1}' with message '{2}'.</value>
+    <value>Analyzer '{0}' threw an exception of type '{1}' with message '{2}'.
+{3}</value>
```

Comment:
Best practice is to document the placeholders in the notes field of each item; this helps during the translation process.

Reply:
Is there a reference document regarding standard documentation for common types of placeholders?